features/admin.branding.ai.v1/components/branding-ai-banner.tsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import SendOutlinedIcon from "@mui/icons-material/SendOutlined";
+import { Collapse } from "@mui/material";
+import Box from "@oxygen-ui/react/Box";
+import Chip from "@oxygen-ui/react/Chip";
+import CircularProgress from "@oxygen-ui/react/CircularProgress";
+import IconButton from "@oxygen-ui/react/IconButton";
+import TextField from "@oxygen-ui/react/TextField";
+import Typography from "@oxygen-ui/react/Typography";
+import {
+    DocumentationLink,
+    useDocumentation
+} from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import AIBanner from "../../common.ai.v1/components/ai-banner";
+import AIBannerTall from "../../common.ai.v1/components/ai-banner-tall";
+import useAIBrandingPreference from "../hooks/use-ai-branding-preference";
+import useGenerateAIBrandingPreference,
+{ GenerateAIBrandingPreferenceFunc } from "../hooks/use-generate-ai-branding-preference";
+import { BannerState } from "../models/types";
+import "./branding-ai-banner.scss";
+
+/**
+ * Branding AI banner component.
+ */
+export const BrandingAIBanner: FunctionComponent = (): ReactElement => {
```

Comment:
Shall we use default exports?

Reply:
Fixed with 2b12f3be4e49de7756b36bf0797366b0376b6b10